src/rpc/blockchain.cpp

```diff
@@ -351,7 +351,8 @@ static UniValue getdifficulty(const JSONRPCRequest& request)
 
 static std::string EntryDescriptionString()
 {
-    return "    \"size\" : n,             (numeric) virtual transaction size as defined in BIP 141. This is different from actual serialized size for witness transactions as witness data is discounted.\n"
+    return "    \"vsize\" : n,            (numeric) virtual transaction size as defined in BIP 141. This is different from actual serialized size for witness transactions as witness data is discounted.\n"
+           "    \"size\" : n,             (numeric) same as vsize (DEPRECATED)\n"
```

Comment:
I think this should document that `size` is only returned if `deprecatedrpc=size` is in the config or passed as an argument to bitcoind. Perhaps:

```
           "    \"size\" : n,             (numeric) (DEPRECATED) same as vsize. Only returned if bitcoind is started with -deprecatedrpc=size\n"
           "                              `size` will be completely removed in v0.18.\n"
```